src/main/java/org/folio/rest/impl/EholdingsPackagesImpl.java

```diff
@@ -247,9 +247,9 @@ public void deleteEholdingsPackagesByPackageId(String packageId, Map<String, Str
           rmapiConfiguration.getUrl(), vertxContext.owner()));
         return rmapiService.getValue().retrievePackage(parsedPackageId);
       })
-      .thenCompose( packageData -> {
-        if(!packageData.getIsCustom()){
-          throw new InputValidationException(INVALID_PACKAGE_TITLE, INVALID_PACKAGE_DETAILS);
+      .thenCompose(packageData -> {
+        if (!packageData.getIsCustom()) {
+          throw new InputValidationException(INVALID_PACKAGE_DETAILS, INVALID_PACKAGE_TITLE);
```

Comment:
Instead of swapping the variables, can we just swap error messages instead, the place they are assigned to these variables? Thanks!

Reply:
Thanks, updated